test/common/storagelib.py

```diff
@@ -80,16 +81,15 @@ def add_loopback_disk(self, size=50, name=None):
         # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1969408
         # It would be nicer to remove $F immediately after the call to
         # losetup, but that will break some versions of lvm2.
-        dev = self.machine.execute("F=$(mktemp /var/tmp/loop.XXXX); "
-                                   "truncate --size=%sMB $F; "
-                                   "losetup --show %s $F" % (size, name if name else "--find")).strip()
+        temp_path = f"/var/tmp/loop.{time.time()}"
+        dev = self.machine.execute(f"truncate --size={size}MB {temp_path}; losetup --show {name if name else '--find'} {temp_path}").strip()
         # If this device had partions in its last incarnation on this
         # machine, they might come back for unknown reasons, in a
         # non-functional state. Running partprobe will get rid of
         # them.
         self.machine.execute("partprobe '%s'" % dev)
         # right after unmounting the device is often still busy, so retry a few times
-        self.addCleanup(self.machine.execute, f"umount {dev} || true; rm $(losetup -n -O BACK-FILE -l {dev}); until losetup -d {dev}; do sleep 1; done", timeout=10)
+        self.addCleanup(self.machine.execute,f"findmnt --noheadings --output=TARGET {dev} | xargs --no-run-if-empty umount; until losetup -d {dev}; do sleep 1; done; rm {temp_path}""", timeout=10)
```

Comment:
Still the same two issues from above -- extra `""` (see test/static-code failure) and too long line

Reply:
Sorry, I've missed this issue, as I've tested it locally and then just changed.